var/spack/repos/builtin/packages/py-isort/package.py

```diff
@@ -20,8 +21,10 @@ class PyIsort(PythonPackage):
 
     variant("colors", default=False, description="Install colorama for --color support")
 
-    depends_on("python@3.6.1:3", type=("build", "run"), when="@5:")
-    depends_on("python@2.7:2.8,3.4:", type=("build", "run"), when="@4.3:")
-    depends_on("python@2.6:2.8,3.3:", type=("build", "run"))
+    depends_on("python@3.8:", when="@5.12:", type=("build", "run"))
+    depends_on("python@3.6.1:3", when="@5:5.10", type=("build", "run"))
     depends_on("py-poetry-core@1:", type="build")
-    depends_on("py-colorama@0.4.3:0.4", type=("build", "run"), when="+colors")
+    depends_on("py-colorama@0.4.3:", when="+colors @5.12:", type=("build", "run"))
+    depends_on("py-colorama@0.4.3:0.4", when="+colors @:5.11", type=("build", "run"))
+
+    conflicts("^py-poetry-core@1.5:", when="@:5.11")
```

Comment:
```suggestion
    conflicts("^py-poetry-core@1.5:", when="@:5.11.4")
```

Reply:
Also can you add a comment linking to the issue?

Reply:
Done